fl-server/src/auth.rs

```diff
@@ -78,7 +78,7 @@ pub async fn sign_in_handler(
     }))
 }
 
-fn get_user_by_username(users: Vec<User>, username: &str) -> Option<User> {
+pub fn get_user_by_username(users: Vec<User>, username: &str) -> Option<User> {
```

Comment:
In general, why do u need to return a copy of the user not a &? only copy when u actually need to. But you can easily return a ref to the user in the vec and use it